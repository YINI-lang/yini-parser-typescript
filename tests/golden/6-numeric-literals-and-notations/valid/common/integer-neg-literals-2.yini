// Common negative edge-case decimal integers.
^ NegInts
int1 = -0
int2 = -1                     // smallest negative integer
int3 = -2                     // just below -1
int4 = -2147483648            // min 32-bit signed integer
int5 = -9223372036854775808   // min 64-bit signed integer
int6 = -999999999             // large negative but still within 32-bit range
