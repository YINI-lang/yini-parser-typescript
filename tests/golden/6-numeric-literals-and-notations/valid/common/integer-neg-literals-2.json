{
    "NegInts": {
        "int1": 0,
        "int2": -1,
        "int3": -2,
        "int4": -2147483648,
        "int5": -9223372036854775808,
        "int6": -999999999
    }
}
